docs/docs-content/clusters/edge/local-ui/host-management/theming.md

```diff
@@ -78,18 +80,19 @@ console for all Edge hosts of your organization.
        "brand": "#4A8FF1",
        "sidebar": "#2B323C"
      },
-     "logo": "/logo.webp"
+     "logo": "logo.webp"
    }
    ```
 
-   The sidebar color controls the color of the sidebar. This is also the color of the background in the Local UI login
-   screen. We suggest you choose a color that contrasts well against your logo as the logo as the color will serve as
-   the background for your logo. The **brand** color controls the color of buttons and checkboxes in the UI. The
+   The `colors.sidebar` controls the color of the sidebar. This is also the color of the background in the Local UI
+   login screen. We suggest you choose a color that contrasts well against your logo as the logo as the color will serve
+   as the background for your logo. The **brand** color controls the color of buttons and checkboxes in the UI. The
```

Comment:
```suggestion
   The `colors.sidebar` property controls the color of the sidebar. This is also the color of the background in the Local UI
   login screen. We suggest you choose a color that contrasts well against your logo as the logo as the color will serve
   as the background for your logo. The **brand** color controls the color of buttons and checkboxes in the UI. The
```

Reply:
> We suggest you choose a color that contrasts well against your logo as the logo as the color will serve as the background for your logo.
   
I think something was lost here 🤔 Maybe, _We suggest you choose a color that contrasts well against your logo, because this color will also serve as the logo background._

Reply:
Also, we could maybe suggest a tool like [WebAIM](https://webaim.org/resources/contrastchecker/) to check the contrast and recommend aiming for AA at least?